### ISSUE TITLE
More issues of spurious spaces

### ISSUE THREAD

**Issue by Frenchomatic** [ahmedkaludi/accelerated-mobile-pages#2403]
It seems we have really taken a backward step with how html is being converted from the editor. 

Example code 

`<h2 class="bottom-shadow">Raid à ski dans <span class="greener">les Cerces</span></h2>
`
gets rendered as 

`<h2 class="bottom-shadow">Raid à ski dans <span class="greener">les Cerces</span>SPACE</h2>`

The css for bottom shadow which is simple underlining of the H2  

```
.bottom-shadow {
padding: 10px 0px;
display: inline-block;
position: relative;
}

.bottom-shadow:before {
content: "";
position: absolute;
width: 30%;
height: 1px;
bottom: 0;
border-bottom: 3px solid #5aaedc;
}

.greener {
color: #8bbd49;
}
```

This appears to occur with minification set to on. Also just so no confusion there is no W3 total cache or anything else. Again it is code where you have `><` as in this case `</span></h2>`

**Comment by Zabi09** [comment]
@Frenchomatic 

I totally understand your concern about the spaces but can you please tell me that is these spaces are affecting the output of it because I recreated the same on my end and the CSS and heading are appearing perfectly without any issues - http://take.ms/6gRkF

Please let me know.

**Comment by Frenchomatic** [comment]
Have a look in the actual source code that it generates and let me know if you get the problem. When I turn minification off the space disappears.

**Comment by Frenchomatic** [comment]
And if I roll back to a previous version Version 0.9.96.1 for example this bug doesn't exist.

**Comment by Frenchomatic** [comment]
see #2406

**Comment by merajali294** [comment]
@Zabi09, we need to fix this issue.

**Comment by Zabi09** [comment]
This will be fixed in #2445
